src/Standards/Squiz/Tests/Arrays/ArrayDeclarationUnitTest.1.inc

```diff
@@ -486,3 +486,20 @@ $foo = [
 $a = array(
       'a' =>
      );
+
+// Ensure absence of NoKeySpecified error for PHP 7.4+ array unpacking.
```

Comment:
This comment only refers to the one error code, while both are now handled. Maybe something like this ?
```suggestion
// Safeguard correct errors for key/no key when PHP 7.4+ array unpacking is encountered.
```